CONTRIBUTING.md

```diff
@@ -667,20 +666,15 @@ Below is the list of available flags:
     This applies only to tests that enable connectInject.
 -enterprise-license
     The enterprise license for Consul.
--kubeconfig string
-    The path to a kubeconfig file. If this is blank, the default kubeconfig path (~/.kube/config) will be used.
--kubecontext string
-    The name of the Kubernetes context to use. If this is blank, the context set as the current context will be used by default.
--namespace string
-    The Kubernetes namespace to use for tests. (default "default")
+-kubeconfigs string
+    The comma separate list of Kubernetes configs to use (eg. "~/.kube/config,~/.kube/config2"). The first in the list will be treated as the primary config, followed by the secondary, etc. If this is empty, or fields are blank, then the context set as the current context will be used by default.
+-kubecontexts string
+    The comma separate list of Kubernetes contexts to use (eg. "kind-dc1,kind-dc2"). The first in the list will be treated as the primary context, followed by the secondary, etc. If this is empty, or fields are blank then the default kubeconfig path (~/.kube/config) will be used.
+-kubenamespaces string
+    The comma separate list of Kubernetes namespaces to use (eg. "consul,consul-secondary"). The first in the list will be treated as the primary namespace, followed by the secondary, etc. If the list is empty, or fields are blank, then the "default" namespace will be used.
```

Comment:
We don't use namespaces anywhere from what I can tell

Reply:
We should use the default namespace from your context instead (this is how it works today). So if I call `kubectl config set-context --current --namespace=some-other-namespace`, we should use `some-other-namespace` instead of `default`. This is also how all of kube machinery works.

Reply:
We don't set kubenamespaces explicitly anywhere in the pipeline. For openshift tests, we used to set it in the context in the kubeconfig file because you don't want to use the default project on openshift.

Reply:
ok, I can verify that works as before

Reply:
verified that if the namespace isn't provided it uses whatever you set like in the command Iryna posted:
`kubectl config set-context --current --namespace=some-other-namespace`

Reply:
Just verified that this doesn't work as expected, this didn't work before either (I checked out main) so we'll have to create an issue and fix this at some point.

Nothing is really exercising the behaviour currently so I guess it broke at some point